node/pkg/driver/device_connectivity/device_connectivity_iscsi.go

```diff
@@ -32,6 +36,44 @@ func NewOsDeviceConnectivityIscsi(executer executer.ExecuterInterface) OsDeviceC
 	}
 }
 
+func (r OsDeviceConnectivityIscsi) iscsiCmd(args ...string) (string, error) {
+	out, err := r.Executer.ExecuteWithTimeout(int(iscsiCmdTimeout.Seconds()), "iscsiadm", args)
+	return string(out), err
+}
+
+func (r OsDeviceConnectivityIscsi) iscsiDiscoverAndLogin(targetName, targetPortal string) error {
+	logger.Debugf("iscsiDiscoverAndLogin: target: {%s}, portal: {%s}", targetPortal, targetPortal)
+	output, err := r.iscsiCmd("-m", "discovery", "-t", "sendtargets", "-p", targetPortal)
```

Comment:
You can use -o new to add the record first, but usually we only run this command
`iscsiadm -m discoverydb -t st -p port_ip:3260 --discover`